solution/build.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko
+PRODUCT_LOWERNAME=zenko
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=unix:///var/run/docker.sock
+HARDLINK=hardlink
+OPERATOR_SDK=operator-sdk
+OPERATOR_SDK_OPTS=
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+
+# grab our dependencies from our deps.txt file as an array
+readarray -t DEP_IMAGES < deps.txt
+# add the CR dependencies by joining image: and tag: lines
+readarray -t CR_IMAGES < <(grep image: zenko-version-cr.yaml | awk '{print $2}')
+readarray -t CR_TAGS < <(grep tag: zenko-version-cr.yaml | awk '{print $2}')
+
+for index in ${!CR_IMAGES[@]}; do
+	DEP_IMAGES+=(${CR_IMAGES[index]}:${CR_TAGS[index]})
+done
+
+function clean()
+{
+	rm -rf ${BUILD_ROOT}
+}
+
+function mkdirs()
+{
+	mkdir -p ${ISO_ROOT}
+}
+
+function copy_yamls()
+{
+	OPERATOR_TAG=$(grep /zenko-operator: deps.txt | awk -F ':' '{print $2}')
+	sed "s/ZENKO_OPERATOR_TAG/${OPERATOR_TAG}/" config.yaml > ${ISO_ROOT}/config.yaml
+	cp -R -f operator/ ${ISO_ROOT}/operator
+}
+
+function gen_product_txt()
+{
+	cat > ${ISO_ROOT}/product.txt <<EOF
+NAME=${PRODUCT_NAME}
+VERSION=${VERSION_FULL}
+SHORT_VERSION=${VERSION_SHORT}
+GIT=${GIT_REVISION}
+DEVELOPMENT_RELEASE=1
+BUILD_TIMESTAMP=${BUILD_TIMESTAMP}
+BUILD_HOST=${BUILD_HOST}
+EOF
+}
+
+function gen_operator_yaml()
+{
+	OPERATOR_IMAGE=$(grep /zenko-operator: deps.txt)
+	# we need to escape / with \/ in our sed command
```

Comment:
(or use a different separator, e.g. `sed "s|REPLACE_IMAGE|${OPERATOR_IMAGE}|" operator.yaml`)

Reply:
oh my comment doesnt actually apply anymore... but good to know.